src/buildSummary.ts

```diff
@@ -43,20 +43,26 @@ export function processAndDisplayBuildSummary() {
 
     const filePath: string = join(runnerTemp, `buildSummary${runId}.json`);
     let taskSummaryTableRows;
-    try {
-        const data = JSON.parse(readFileSync(filePath, { encoding: 'utf8' }));
-        taskSummaryTableRows = getBuildSummaryTable(data);
-    } catch (e) {
-        console.error('An error occurred while reading the build summary file:', e);
-        return;
-    } finally {
+    if (checkFileExists(filePath)) {
```

Comment:
Can use import and use `fs.existsSync` and remove the custom `checkFileExists` definition. Can also remove `accessSync` from import list.
```suggestion
    if (existsSync(filePath)) {
```

Reply:
I saw exist deprectaed in chat gpt hence did not use it :) shall update it